package.json

```diff
@@ -30,7 +30,8 @@
     "JSONStream": "^1.2.0",
     "async": "^2.0.1",
     "aws4": "^1.4.1",
-    "awscred": "^1.2.0",
+    "aws-sdk": "^2.36.0",
+    "path": "^0.12.7 ",
```

Comment:
@evantahler elasticdump currently uses this dependency as a transitive dependency of awscred (which I am removing).

https://github.com/mhart/awscred/blob/master/index.js#L2

It is in used in aws4signer.js:35 to keep compatibility with the way that elasticdump currently pulls creds from a config file. Seems like path is a helper library to make sure paths are formed in a platform-safe manner. Is there a better/recommended way you'd prefer?

aws4signer.js:35
```javascript
path.join(os.homedir(), '.aws', parent.options.awsIniFileName ? parent.options.awsIniFileName : 'config')
```